Doc/library/stdtypes.rst

```diff
@@ -2920,9 +3211,8 @@ arbitrary binary data.
    Split the sequence at the first occurrence of *sep*, and return a 3-tuple
    containing the part before the separator, the separator itself or its
    bytearray copy, and the part after the separator.
-   If the separator is not found, return a 3-tuple
-   containing a copy of the original sequence, followed by two empty bytes or
-   bytearray objects.
+   If the separator is not found, return a 3-tuple containing a copy of the original sequence,
+   followed by two empty bytes or bytearray objects.
 
```

Comment:
In general, please avoid rewrapping, especially with no other changes, it makes it harder to review.

```suggestion
   If the separator is not found, return a 3-tuple
   containing a copy of the original sequence, followed by two empty bytes or
   bytearray objects.
```